circuit/Cargo.toml

```
@@ -50,3 +50,14 @@ version = "0.9.12"
 [dependencies.snarkvm-circuit-types]
 path = "./types"
 version = "0.9.12"
+
+[features]
+default = [ ]
```

I think enabling `parallel` by default makes sense, so that we benefit by default from the efficiency benefits of parallelism, but still preserve the ability to turn off `parallel`.

```suggestion
default = [ "parallel" ]
```

---

I took a stab at implementing your suggestions and I suggest we don't continue in that direction: https://github.com/demox-labs/snarkVM/commit/7ead9e29e2af932d53f5d65e93bbb0febfb907b8

This is what a serial feature would look like:
https://github.com/demox-labs/snarkVM/commit/ebf9c6234199e7ca5e849a762a72b2e145fbc793